sdk/core/System.BinaryData/src/BinaryData.cs

```diff
@@ -0,0 +1,260 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    /// <summary>
+    /// A lightweight abstraction for a payload of bytes that supports converting between string, stream, Json, and bytes.
+    /// </summary>
+    public class BinaryData
+    {
+        private const int CopyToBufferSize = 81920;
```

Comment:
this seems large. how did we come up with this number?

Reply:
This was copied from similar usages in our repo - https://github.com/Azure/azure-sdk-for-net/search?q=81920

Reply:
@tg-msft, @pakrym, do you know where the size came from? It seems large too me.

Reply:
https://github.com/dotnet/runtime/blob/85162fbf9ccdeb4fa1df357f27308ae96579c066/src/libraries/System.Private.CoreLib/src/System/IO/Stream.cs#L33